README.Mac.md

```diff
@@ -1,294 +1,281 @@
-# LORIS Mac Install Guide
-### Mac is no longer supported as of 15.10. 
+### These instructions last tested for LORIS version v20.*
+# Install a developer instance of LORIS on macOS
+### We recommend installing LORIS on Ubuntu or CentOS
+#### macOS is no longer supported for "production" as of v15.10. 
 
-This is a guide on how to get a LORIS instance Running on your Mac computer. It is designed for installation in a sandbox environment. 
-It has been partially tested for Mac OS X 10.9 and 10.10 for 15.04. Installing on Mac OS X El Capitan will require additional Apache setup. 
-For best results, we recommend installing LORIS on Ubuntu or CentOS.
+This is a guide on how to get a LORIS instance running on your Apple computer for development. You're expected to have basic understanding of using a terminal and editing configuration files. 
+For production, we recommend installing LORIS on Ubuntu or CentOS.
 
 # 1. System Requirements
 
-Before continuing to the subsequent sections ensure that the following tools are installed.  Sudo permission is required.  This section covers installation and recommended versions of:
+The following list provides foresight into the proceeding sections and recommended versions to install:
+
+ * macOS (tested on Mojave 10.14.3)
+ * XCode Command Line Tools
+ * Homebrew
+ * Apache (a.k.a httpd)
```

Comment:
If it's only for development as described, I don't see why this is using apache. The setup would be simplified and less error prone by just using `php -S` like Travis does.

Reply:
@driusan can you explain further because I'm unaware of php -S. This is the configuration I've been using to develop with Loris without using a VM. Using apache I thought was necessary with viewing the error logs and making it as close to a production environment at the same time.

Reply:
`php -S` is php's built in development web server. I used `php -S localhost:8000 -t . router.php` from the htdocs directory (logs go to stderr).. it's not as close to production, but it's much easier to setup and it's closer to Travis

Reply:
@driusan can I leave apache in the guide but add an optional tag. Also adding the step for php -S because there are benefits to both cases.

Reply:
@driusan I added php -S as an alternative because I think both are valuable.